databricks/koalas/groupby.py

```diff
@@ -2328,6 +2334,8 @@ def _apply_series_op(self, op, should_resolve: bool = False):
         applied = []
         for column in self._agg_columns:
             applied.append(op(column.groupby(self._groupkeys)))
+        if not applied and (op.__name__).startswith("cum"):
+            raise DataError("No numeric types to aggregate")
```

Comment:
Please don't rely on the function name but add an explicit argument, as `Frame._reduce_for_stat_function` has `numeric_only=True`.

Reply:
@ueshin Thanks for the advice. I'll manage it 😃